### DISCUSSION TITLE
Improve the hero section design

### DISCUSSION THREAD

**Discussion by Luson045** [Luson045/medi-connect#109] [Announcements]
<img width="960" alt="image" src="https://github.com/user-attachments/assets/ec955c7a-5865-4f06-8934-eb965da29f0d">
This is our current landing page, as you can see it looks like some 90's webpage.

Take your first step to improve the design of the hero section, add catchy phrases, use better colours and images to make it look attractive as well as keep it simple.
Make sure that it is responsive.

Good PR will be accepted as soon as possible.

**Comment by kartikey369-ind** [comment]
Hi @Luson045 ,

I hope you’re doing well! As we work on improving our open-source project, I’d like to assign you the task of redesigning the hero section of our landing page.

what I'll do---->
Modernize the hero section to make it visually appealing and user-friendly.
Key Elements to Include:
Create catchy phrases that reflect our mission.
Choose a modern color scheme that enhances the design.
Incorporate high-quality images or illustrations relevant to our project.
Ensure the design is responsive for various devices.


hope you assign this work to me.if possible could you please update the label to gssoc24-extd as iam working on it even

**Comment by Luson045** [comment]
Are you familiar with react?

**Comment by kartikey369-ind** [comment]
I am a newbie in programming

**Comment by kartikey369-ind** [comment]
i can use html,css and js if it works for you

**Comment by Luson045** [comment]
okay give it a try, if you need any help, then feel free to ask 👍

**Comment by kartikey369-ind** [comment]
Ok sure sent you linkedin connection request 
(Kartikey korimilli)

**Comment by Luson045** [comment]
@SuperexMack would you like to work on the Home page issue(#6), i would label it as hacktoberfest ans gssoc, as one guy is already working on it

**Comment by Jayesh-Waghmare** [comment]
@Luson045  I am interested in working on this issue. Please could you assign it to me?

**Comment by Luson045** [comment]
@Jayesh-Waghmare well i think @kartikey369-ind is taking care of this one so can you take some other page, as we have decided to redesign most of the pages

**Comment by Luson045** [comment]
@kartikey369-ind i want some updates on this by the end of tomorrow else I will assign it to other contributors

**Comment by kartikey369-ind** [comment]
Iam sorry @Luson045 iam unable to do it so please assign it to someone else

**Comment by Luson045** [comment]
@kartikey369-ind okay 👍, you can find some other issues that suits you, start small..
